index.js

```diff
@@ -11,13 +11,16 @@ var moderate = 'color:orange;font-weight:bold;';
 var minor = 'color:orange;font-weight:normal;';
 var defaultReset = 'font-color:black;font-weight:normal;';
 
-var timer;
+var handler;
 var timeout;
 var _createElement;
 var components = {};
 var nodes = [];
 var cache = {};
 
+var scheduleFunction = requestIdleCallback || setTimeout;
```

Comment:
This will raise a `ReferenceError: requestIdleCallback is not defined` in environments that do no have `requestIdleCallback`.

We should either pad this more defensively, or use a polyfill (eg [`npmjs.com/requestidlecallback`](https://www.npmjs.com/package/requestidlecallback)).